source/docs/api-reference/wpilib-integration/gyro-integration.rst

```diff
@@ -0,0 +1,20 @@
+Gyro Integration
+================
+
+CTR-Electronics supported gyroscopes (Pigeon 2) implement the WPILib ``Gyro`` (`Java <https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/interfaces/Gyro.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/release/cpp/classfrc_1_1_gyro.html>`__) interface. Phoenix Pro implements all inherited functions:
+
+- ``reset``
+- ``getAngle``
+- ``getRate``
+- ``getRotation2d``
+
+.. note:: ``calibrate()`` is a no-op (does nothing) since the Pigeon 2.0 performs calibration on boot.
+
+Simulation
+----------
+
+All signal states are shown in the simulation :guilabel:`Other Devices` menu.
```

Comment:
Same as in motor controller integration, I don't like the wording of "All signal states are shown", maybe "The simulated device state is shown"?